packages/jest-changed-files/src/git.js

```diff
@@ -40,9 +40,11 @@ function findChangedFiles(
         if (stdout === '') {
           resolve([]);
         } else {
-          resolve(stdout.split('\n').map(
-            changedPath => path.resolve(cwd, changedPath),
-          ));
+          resolve(
+            stdout
+              .split('\n')
+              .map(changedPath => path.resolve(cwd, changedPath)),
```

Comment:
For this one, unfortunately I don't think it's possible to get it correct all the time. if `stdout` was a string then you'd likely want to write it that way.